parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -295,6 +295,13 @@ ensureContainerd() {
   systemctl is-active --quiet docker && (systemctl_disable 20 30 120 docker || exit $ERR_SYSTEMD_DOCKER_STOP_FAIL)
   systemctlEnableAndStart containerd || exit $ERR_SYSTEMCTL_START_FAIL
 }
+{{- if and IsKubenet (not HasCalicoNetworkPolicy)}}
+ensureNoDup() {
```

Comment:
nit: NoDup seems too vague, maybe ensureNoDupOnPromiscuBridge? actually is it possible to skip this for non-promiscuous mode, in case we get the kernel fix and use hairpin-veth mode?

Reply:
Hm yea, I can add a check in the service script itself to exit 0 early if the bridge is not in promiscMode.